pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/equation/containers/SingleValueVector.java

```diff
@@ -0,0 +1,30 @@
+package org.broadinstitute.ddp.equation.containers;
+
+
+import lombok.AllArgsConstructor;
+import lombok.EqualsAndHashCode;
+
+import java.math.BigDecimal;
+import java.util.Collections;
+import java.util.List;
```

Comment:
These imports should go above the lombok imports. While it's not an error, Checkstyle does expect the import ordering to be:
* static imports
* Java/javax
* All others